### PR TITLE
Add script to checkout all repos and do some cleaning

### PR DESCRIPTION
**Pull request by hammady**
- Add checkout rayyan repos script
- Deprecate git submodules, docker-compose, swarmmode, rayyansolr, dockertest and fluentd
- Update README with a link to the wiki

### CODE REVIEW

File: checkout-rayyan-repos.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+set -e
+
+if [ $# -ne 1 ]; then
+  echo "$USAGE: $0 <checkout-dir>"
+  exit 1
+fi
+
+basedir=$1
+if [ -d $basedir ]; then
```
Comment (fabubaker) [inline checkout-rayyan-repos.sh:10]: So I can't do `./checkout-rayyan-repos.sh .` or `./checkout-rayyan-repos.sh ../`?

File: checkout-rayyan-repos.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    echo 'kubectl create namespace rayyan-dev --dry-run -o yaml | kubectl apply -f -'
+) > $skaffold_tmpfile
+cleanskaffold_tmpfile=$(mktemp /tmp/rayyan-skaffold.XXXXXX)
+(
+    echo '#!/usr/bin/env bash'
+) > $cleanskaffold_tmpfile
+
+# checkout open source repos
+for repo in \
+    rayyan-cloud \
```
Comment (fabubaker) [inline checkout-rayyan-repos.sh:32]: Considering rayyan-cloud has to be cloned to run this script anyways, should we clone it again here?

Comment (hammady) [reply checkout-rayyan-repos.sh:32]: No it doesn't have to. The [wiki](https://dev.azure.com/qcrisw/Rayyan/_wiki/wikis/Rayyan.wiki/79/Getting-started?anchor=get-the-code) has a `curl` command to just grab this script and run it which will consequently clone all repos, including this one.

File: checkout-rayyan-repos.sh (hunk 1 above)
Comment (hammady) [reply checkout-rayyan-repos.sh:10]: No you can't. This is a conservative decision as those dirs might have dirs with the same name of the repos.

**Review by fabubaker — APPROVED** [review]

**Pull request merged**